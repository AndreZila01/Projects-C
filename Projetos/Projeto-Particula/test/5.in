RJ A
RE A
RP A 1
0 1
0 0 0
0 0 0
RP A 1
0 -1
1 1 1
0 0 0
S A 1 1
5 1
-