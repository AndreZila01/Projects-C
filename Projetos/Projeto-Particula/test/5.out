Utilizador registado com sucesso.
Espaço de simulação registado com identificador 1.
Partícula registada com identificador 1.
Partícula registada com identificador 2.
t,px,py,pz,vx,vy,vz,ax,ay,az,fgx,fgy,fgz,fex,fey,fez,fx,fy,fz
0,0.000E+00,0.000E+00,0.000E+00,0.000E+00,0.000E+00,0.000E+00,0.000E+00,0.000E+00,0.000E+00,0.000E+00,0.000E+00,0.000E+00,8.988E+09,8.988E+09,8.988E+09,   INF,   INF,   INF
1,4.494E+09,4.494E+09,4.494E+09,8.988E+09,8.988E+09,8.988E+09,8.988E+09,8.988E+09,8.988E+09,0.000E+00,0.000E+00,0.000E+00,-4.450E-10,-4.450E-10,-4.450E-10,  -INF,  -INF,  -INF
2,1.348E+10,1.348E+10,1.348E+10,8.988E+09,8.988E+09,8.988E+09,-4.450E-10,-4.450E-10,-4.450E-10,0.000E+00,0.000E+00,0.000E+00,-4.945E-11,-4.945E-11,-4.945E-11,  -INF,  -INF,  -INF
3,2.247E+10,2.247E+10,2.247E+10,8.988E+09,8.988E+09,8.988E+09,-4.945E-11,-4.945E-11,-4.945E-11,0.000E+00,0.000E+00,0.000E+00,-1.780E-11,-1.780E-11,-1.780E-11,  -INF,  -INF,  -INF
4,3.146E+10,3.146E+10,3.146E+10,8.988E+09,8.988E+09,8.988E+09,-1.780E-11,-1.780E-11,-1.780E-11,0.000E+00,0.000E+00,0.000E+00,-9.082E-12,-9.082E-12,-9.082E-12,  -INF,  -INF,  -INF
